shared/src/test/scala/singleton/ops/ImplicitConversionSpec.scala

```diff
@@ -11,5 +11,5 @@ class ImplicitConversionSpec extends Properties("ImplicitConversion") {
   property("Wrong OpToOp") = {illTyped("""val ret : 2 + 1 = implicitly[4 - 2]"""); true}
   property("Wrong OpToSingleton") = {illTyped("""val ret : 3 = implicitly[4 - 2]"""); true}
   property("Wrong SingletonToOp") = {illTyped("""val ret : 2 + 1 = 4"""); true}
-  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueOf[4 - 1]}
-}
\ No newline at end of file
+  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueof.valueOf[4 - 1]}
```

Comment:
Sounds good but I won't be able to do it until tomorrow. If you need to merge #33 now feel free to do so and i'll rebase later